DSCResources/MSFT_xPowerPlan/MSFT_xPowerPlan.psm1

```diff
@@ -0,0 +1,108 @@
+function Get-TargetResource
+{
+    [CmdletBinding()]
+    [OutputType([System.Collections.Hashtable])]
+    param
+    (
+        [Parameter(Mandatory = $true)]
+        [ValidateSet('Yes')]
+        [System.String]
+        $IsSingleInstance,
```

Comment:
@eshaparmar I could add # This is the best practice when writing a single-instance DSC resource. Would that suffice?